docs/en/sql-reference/functions/string-functions.md

```diff
@@ -1150,3 +1150,13 @@ A text with tags .
 The content within <b>CDATA</b>
 Do Nothing for 2 Minutes 2:00 &nbsp;
 ```
+
+## ascii(s) {#ascii}
+
+Returns the ASCII code point of the first character of str.  The result type is Int32.
+
+If s is empty, the result is 0. If the first character is not an ASCII character or part of the Latin-1 Supplement range of UTF-16, the result is undefined.
```

Comment:
Slight ambiguity. It should say: "If the first character is not an ASCII character or **not** part of the Latin-1 Supplement range of UTF-16, [...]"

Also: "If **str** is empty, [....]"